pom.xml

```diff
@@ -1397,6 +1399,17 @@
     </profile>
 
     <profile>
+      <id>hbase-1.0</id>
```

Comment:
Apologies for letting this PR keep opened.

Profiles and Properties for specific interpreter is now moved to interpreter's pom.xml. 
It'll be helpful rebase this branch and move Profiles/Properties into hbase/pom.xml.